apps/wallet-mobile/src/Receive/ReceiveScreen.tsx

```diff
@@ -41,63 +42,57 @@ export const ReceiveScreen = () => {
 
         <Content>
           <View style={styles.address}>
-            {currentAddress != null ? (
-              <AddressDetail address={currentAddress} />
+            {currentAddress !== null ? (
+              <AddressDetail address={currentAddress} title={strings.addresscardTitle} />
             ) : (
-              <ActivityIndicator size="large" color="black" />
+              <SkeletonAdressDetail />
             )}
           </View>
 
-          <Spacer height={24} />
+          <Spacer height={64} />
 
           <Button
-            outlineOnLight
-            onPress={() => wallet.generateNewReceiveAddress()}
-            disabled={addressLimitReached}
-            title={!addressLimitReached ? strings.generateButton : strings.cannotGenerate}
-            testID="generateNewReceiveAddressButton"
+            outline
+            title='request specific amount'
+            textStyles={{
+              color: colors.buttonBackgroundBlue
+            }}
           />
 
           <Spacer height={24} />
 
-          <UnusedAddresses />
-
-          <Spacer height={24} />
+          <Button
+            shelleyTheme
+            onPress={() => {
+              copy(currentAddress)
+            }}
+            disabled={currentAddress === null ? true : false}
+            testID="copyReceiveAddressButton"
+            title='Copy address'
+            iconImage={require('../../src/assets/img/copy.png')}
```

Comment:
We avoid png images. Could you use the svg and transform it to react native component using this tool: https://transform.tools/svg-to-react-native